DataAccess/DataAccess.Postgre/PostgreRepositories.cpp

```diff
@@ -473,30 +506,6 @@ std::optional<pqxx::result> RepliesRepository::insertIDsIntoChannelRepliesTable(
     return pTable->Insert()->columns(dataForChannelReplies)->returning({"channel_id"})->execute();
 }
 
-Utility::DirectMessageStatus DirectMessageRepository::addDirectChat(uint64_t user_id, uint64_t receiverId)
-{
-    if (user_id == 0) return Utility::DirectMessageStatus::FAILED;
-    pTable->changeTable("channels");
-    auto adapter   = pTable->getAdapter();
-    auto minUserId = std::to_string(std::min(user_id, receiverId));
-    auto maxUserId = std::to_string(std::max(user_id, receiverId));
-    auto result    = adapter->query(
-           "create extension if not exists pgcrypto;\n"
-              "INSERT INTO channels VALUES (DEFAULT, encode(digest(concat(" +
-           minUserId + ", " + maxUserId + "),'sha384'),'base64')," + minUserId + ",2) ON CONFLICT DO NOTHING;");
-
-    result = adapter->query("SELECT id FROM channels WHERE channel_name=encode(digest(concat(" + minUserId + ", " + maxUserId +
-                            "),'sha384'),'base64');");
 
-    if (result.has_value())
-    {
-        auto cId       = std::any_cast<pqxx::result>(result.value());
-        auto channelId = std::to_string(cId[0][0].as<uint64_t>());
-        adapter->query("INSERT INTO user_channels VALUES (" + minUserId + "," + channelId + "), (" + maxUserId + "," + channelId +
-                       ")ON CONFLICT DO NOTHING;");
-        return Utility::DirectMessageStatus::SUCCESS;
-    }
-    return Utility::DirectMessageStatus::FAILED;
-}
 
 }  // namespace DataAccess
```

Comment:
Again, I think there should be no commented out parts of the code that are not used :)